packages/nodes-base/nodes/ReadBinaryFile/ReadBinaryFile.node.ts

```diff
@@ -21,6 +24,7 @@ export class ReadBinaryFile implements INodeType {
 		inputs: ['main'],
 		outputs: ['main'],
 		properties: [
+			allowedPathsNotice,
```

Comment:
I do not think this reasoning is applicable here.
`N8N_RESTRICT_FILE_ACCESS_TO` allows to restrict access to specific folders only e.g. `'/home/user/data;/home/user/documents'`, all other directories would be forbidden
I think that it's reasonable to let user know what location are available for him to read/write

Reply:
I haven't thought much about this, but could we not define a base path for reading/writing binary files from/to? Users could then enter a _relative_ path in their binary file nodes. Or they could enter an _absolute_ path. In both cases, access would only be granted if the resulting path is in compliance with N8N_RESTRICT_FILE_ACCESS_TO.

Reply:
Resolving paths would be somewhat confusing as we basically forbid, in this case, usage of relative paths regarding file system. If we hard code base path for reading/writing we would break functionality for some existing use cases, if we would allow user to define base path it would duplicate `N8N_RESTRICT_FILE_ACCESS_TO`